entry.go

```diff
@@ -208,13 +181,67 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 			return
 		}
 		stream.Close()
+
+		lastComputed := common.Fixed64(0)
+		lastClaimed := common.Fixed64(0)
+		lastUpdate := time.Now()
+		claimInterval := time.Duration(te.Config.ReverseClaimInterval) * time.Second
+		onErr := nkn.NewOnError(1, nil)
+		isClosed := false
+
+		npc, err := te.Wallet.NewNanoPayClaimer(int32(claimInterval/time.Millisecond), onErr, te.Config.ReverseBeneficiaryAddr)
+		if err != nil {
+			log.Println(err)
+			return
```

Comment:
We need to call `te.close()` before return because we can no longer accept payment for this tuna channel.